build/build.sh

```diff
@@ -29,20 +33,23 @@ then
 
     if [[ $TRAVIS_BRANCH == "master" && $(cat version.sbt) != *"SNAPSHOT"* ]]
     then
-        # Run release step only for one build job. Other build jobs in this case do nothing and always succeed.
+        eval "$(ssh-agent -s)"
+        chmod 600 local.deploy_key.pem
+        ssh-add local.deploy_key.pem
+        git config --global user.name "Quill CI"
+        git config --global user.email "quillci@getquill.io"
+        git remote set-url origin git@github.com:getquill/quill.git
+        git fetch --unshallow
+        git checkout master || git checkout -b master
+        git reset --hard origin/master
+
         if [[ $SCALA_VERSION == "2.11" ]]
         then
-            eval "$(ssh-agent -s)"
-            chmod 600 local.deploy_key.pem
-            ssh-add local.deploy_key.pem
-            git config --global user.name "Quill CI"
-            git config --global user.email "quillci@getquill.io"
-            git remote set-url origin git@github.com:getquill/quill.git
-            git fetch --unshallow
-            git checkout master || git checkout -b master
-            git reset --hard origin/master
             git push --delete origin website
```

Comment:
This line fails if there is no `website` branch (this happens if previous build failed after that line but before website was published).
`git push --delete origin website || true` should fix that.

Reply:
@mxl good point!